components/SelectTaskType.tsx

```diff
@@ -18,15 +18,15 @@ export function SelectTaskType(props: { onSelect: (e) => void }): JSX.Element {
   }
   return (
     <div style={{ paddingBottom: 16 }}>
-      <SingleSelect
+      <Autocomplete
         label={"Select type"}
-        items={taskTypes.map((t) => t.name)}
-        handleSelectedItemChange={(e) =>
+        options={taskTypes.map((t) => t.name)}
+        onInputChange={(e) =>
           props.onSelect(
-            taskTypes.find((t) => e.selectedItem === t.name).vsmTaskTypeID //Wow... there must be a better way
+            taskTypes.find((t) => e === t.name).vsmTaskTypeID //Wow... there must be a better way
           )
         }
-        initialSelectedItem={taskTypes[0]?.name}
+        initialSelectedOptions={[taskTypes[0]?.name]}
       />
```

Comment:
This causes TypeScript errors when `strict: true` in `tsconfig.json`.

Reply:
This file is not being used and is deleted in Flyt 2.0. I'm deleting it here as well